bodhi/server/consumers/masher.py

```diff
@@ -658,9 +659,22 @@ def wait_for_mash(self, mash_thread):
         self.log.debug('Waiting for mash thread to finish')
         mash_thread.join()
         if mash_thread.success:
+            notifications.publish(
+                topic='repo.mash.complete',
+                msg=dict(path=self.path,
+                         id=self.id,
+                         agent=self.agent),
+                force=True)
+
             self.state['completed_repos'].append(self.path)
             self.save_state()
         else:
+            notifications.publish(
+                topic='repo.mash.failed',
```

Comment:
Are these ```repo.mash.{complete,failed}``` topics new? I don't see them when git grepping, so I presume they are. It would be good if we add them to these docs as well:

https://fedora-fedmsg.readthedocs.io/en/latest/topics.html#bodhi

Reply:
They are new, yes. And after you agree with the message structure, I would be adding them to fedmsg-meta yes.